src/components/Projects.tsx

```diff
@@ -1,7 +1,149 @@
 import React from "react";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+import stairs from "./assets/project-page/stairs.png";
+import shell from "./assets/project-page/knife.png";
+import eye from "./assets/project-page/eye.png";
+import statue from "./assets/project-page/statue.png";
+import dragon from "./assets/project-page/dragon.png";
+import knife from "./assets/project-page/knife.png";
+import mask from "./assets/project-page/mask.png";
+import spider from "./assets/project-page/spider.png";
+import magnify from "./assets/project-page/magnify.png";
+import Background from "./assets/home-page/background.jpg";
```

Comment:
This has been renamed to "bg" to avoid a conflict with the `${background}` imported from styled-system.  Once I've worked out the fix with the string literal issue, you'll see it referenced on line 96.